opentracing-spring-web-autoconfigure/src/test/java/io/opentracing/contrib/spring/web/autoconfig/TracingAutoConfigurationTest.java

```diff
@@ -0,0 +1,32 @@
+package io.opentracing.contrib.spring.web.autoconfig;
+
+import static org.junit.Assert.assertNotNull;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import io.opentracing.Tracer;
+
+@SpringBootTest(
+        classes = {TracingAutoConfigurationTest.SpringConfiguration.class})
+@RunWith(SpringJUnit4ClassRunner.class)
+public class TracingAutoConfigurationTest {
```

Comment:
Could you please rename it to `TracerAutoConfigurationTest` and test:
* globaltracer is registered
* tracer bean is of the expected type